app/tests/test_queries.py

```diff
@@ -49,9 +53,8 @@ def test_build_fuzzy_search_query_with_results(
 
     def test_build_fuzzy_search_query_no_results(self, client: FlaskClient):
         """
-        Given a filter value that does not match any field used for search terms
-            in any file in the database
-        When build_browse_all_query is called with it and is executed
+        Given a filter value that does not match transferring body in the database
```

Comment:
can you revert back the description to what it was:
```
        Given a filter value that does not match any field used for search terms
            in any file in the database
        When build_browse_all_query is called with it and is executed
```

I updated it when I fixed the flakiness issue